docs/integrations/oneclick.md

```diff
@@ -36,7 +36,8 @@ A primeira chamada será uma teste, para identificar se o seu endpoint existe e
 ```json
 {
   "data_criacao": "2023-07-12T13:58:06.821Z",
-  "evento": "teste_webhook"
+  "evento": "teste_webhook",
```

Comment:
I think that we need to maintain yet, because of the webhook signature, no? When we calculate the HMAC, we calculate with the old fields too.

And on this doc we give an example of cURL passing the hashed signature: https://github.com/Open-Pix/openpix-developers/blob/7c748cc30164cda3402735825245ed1a28b20018/docs/integrations/oneclick.md#L53